2-mandatory/1-weather-report.js

```diff
@@ -12,52 +12,51 @@
 */
 
 function getTemperatureReport(cities) {
-    // TODO
+    let result = [];
+    for (let i = 0; i < cities.length; i++) {
+      result.push(
+        `The temperature in ${cities[i]} is ${temperatureService(
+          cities[i]
+        )} degrees`
+      );
+    }
+    return result;
 }
-
-
 /* ======= TESTS - DO NOT MODIFY ===== */
 
 function temperatureService(city) {
-    let temparatureMap  = new Map();
-
-    temparatureMap.set('London', 10);
-    temparatureMap.set('Paris', 12);
-    temparatureMap.set('Barcelona', 17);
-    temparatureMap.set('Dubai', 27);
-    temparatureMap.set('Mumbai', 29);
-    temparatureMap.set('São Paulo', 23);
-    temparatureMap.set('Lagos', 33);
-    
-    return temparatureMap.get(city);
+  let temparatureMap = new Map();
+
+  temparatureMap.set("London", 10);
+  temparatureMap.set("Paris", 12);
+  temparatureMap.set("Barcelona", 17);
+  temparatureMap.set("Dubai", 27);
+  temparatureMap.set("Mumbai", 29);
+  temparatureMap.set("São Paulo", 23);
+  temparatureMap.set("Lagos", 33);
+
+  return temparatureMap.get(city);
```

Comment:
think your formatter settings conflict with the repo here ; is not a huge deal but something to watch out for